test/metabase/server/middleware/security_test.clj

```diff
@@ -35,37 +35,37 @@
 
 (deftest csp-header-frame-ancestor-tests
   (mt/with-premium-features #{:embedding}
-    (testing "Frame ancestors from `embedding-app-origin` setting"
+    (testing "Frame ancestors from `embedding-app-origins-sdk` setting"
```

Comment:
I think this is a mistake. We want frame ancestors origins to be from `embedding-app-origins-interactive` (or currently `embedding-app-origin`), this doesn't have anything to do with `embedding-app-origins-sdk` which only controls the `Access-Control-Allow-Origin` header.